### ISSUE TITLE
Webpage ?

### ISSUE THREAD

**Issue by eddelbuettel** [ess-intro/ess-intro#9]
Now that we have non-zero videos and slides we probably want a simple webpage linking to them.  

Easiest is probobly a `docs/` folder either in this repo (why not?) or in another repo in the org.  I would suggest a simple markdown file with table, if any of you org-mode pros want an ,org file that works too (but please send a 'build.sh` script or `Makefile` along).

**Comment by aezarebski** [comment]
This sounds like a perfect usecase for [Github pages](https://pages.github.com/), and would mean the site is easy to transition to the ESS homepage or wherever else should that become desirable in the future. 

If the site will remain very simple, org-mode would be a good solution, but if something more sophisticated is desired, it would be better to go with one of the more mainstream solutions, eg Jekyll.

**Comment by eddelbuettel** [comment]
Yes, I use GitHub pages all the time. Transitions are even needed, you can do all that via DNS.  I.e. gallery.rcpp.org is gh-pages repo, and www.rcpp.org is somewhere else.  

But I would vote no to Jekyll (or Hugo).  We are looking at a single page here, really.  Either plain Rmarkdown, or (if we must) org mode.  (I tend to like plain Rmarkdown better).  I recently used `minidown` and its `water` style for several vignettes and tweaked Michael's arm too and helped with an update / refresh of the [Ubuntu CRAN binaries README(s)](https://cloud.r-project.org/bin/linux/ubuntu/).  

And actually as it is github maybe straight up markdown (GitHub style) really is the easiest...

**Comment by aezarebski** [comment]
If it's really only going to be a single page, we may as well just use HTML and avoid the need for any build system. I've made a repo for this which basically says the side is under construction and directs to the github organisation. Easy to scrap it later if requirements change.

https://github.com/ess-intro/ess-intro.github.io

which appears at https://ess-intro.github.io/

**Comment by eddelbuettel** [comment]
Personal view here:   Editing html by hand blows, hard. I still have a few pages around my site I have not converted to markdown.

We're here for ESS. R is a given. Can be **please** agree to (plain, unfancy, no knitr excess) Markdown or RMarkdown and a one liner to call `rmarkdown:: render()`?

**Comment by plantarum** [comment]
I would much prefer {R}markdown to html. Given the built-in support from github, I don't see any advantage to raw html.

**Comment by eddelbuettel** [comment]
There is a minor one.  Github does not render html per se.  You can browse by hand to a md page, and it renders md fine _in its own UI / GitHub site_.  Sure.  

But if, say, we wanted a link off the ESS page itself, we'd need a html server and a html rendering.  GitHub pages does the serving (and @aezarebski has set up all we need) now I think we three are slowly converging on a 'write md, render to html' workflow that should tie the loose ends.

**Comment by eddelbuettel** [comment]
Ok -- made a commit with a simple plain Markdown-as-RMarkdown file and a Makefile.  Certainly not perfect, but maybe closer to good enough? See https://ess-intro.github.io/

**Comment by aezarebski** [comment]
Thanks @eddelbuettel the Rmd solution looks good. I added a `shell.nix` file to make the build easier to reproduce on my machine.

**Comment by eddelbuettel** [comment]
(I usually have a `run.sh` or `runMe.sh` there too.  If I were you I'd keep the `shell.nix` locally and (these days) not commit as likely nobody else is a nix user and I prefer not to clutter repos but it's really marginal...)

**Comment by eddelbuettel** [comment]
We can (should ?) "one day" look into dressing the webpage up.  For now it is good enough, easy to use -- and it is very nice to use the 'org name == repo name' trick for the automagic URL at the org name which I had forgotten about.
